code/controllers/subsystem/tgui.dm

```diff
@@ -309,8 +309,8 @@ SUBSYSTEM_DEF(tgui)
 		return FALSE
 	// Remove it from the list of processing UIs.
 	open_uis.Remove(ui)
-	// If the user exists, remove it from them too.
-	if(ui.user)
+	// If the user exists (and they have open UIs), remove it from them too.
+	if(ui.user.tgui_open_uis)
```

Comment:
Should this be:
``if(ui.user && ui.user.tgui_open_uis)``

Reply:
fixed and fixed another runtime in this